test/export/test_export.py

```diff
@@ -76,7 +76,7 @@ def false_fn(x):
             return x.cos()
 
         def foo(x):
-            return cond(torch.tensor(x.shape[0] > 4), true_fn, false_fn, [x])
+            return torch.ops.cond(torch.tensor(x.shape[0] > 4), true_fn, false_fn, [x])
```

Comment:
When did we add cond to torch.ops? We shouldn't, torch.ops is public API and cond is still experimental.

Reply:
It looks like we didn't add cond to torch.ops, and you're looking for functorch.experimental.control_flow.cond